doc/todo/NEWS.md

```diff
@@ -48,52 +48,63 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 ### <<HIGHLIGHT2>>
 
 
-## Other New Features
-
-We added even more functionality, which could not make it to the highlights:
```

Comment:
A short beginning sentence after every section would be nice. Imho directly starting with lists does not look so nice.

Reply:
> A short beginning sentence after every section would be nice.

Since the sentences basically just describes the heading title I think we can remove them.

Reply:
You can be creative with other sentences :wink:

Reply:
Honestly, I do not really know what do write below the sections that is not just a description of the section title.